atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceImpl.java

```diff
@@ -153,7 +154,7 @@
  * and these inactive nodes will be removed afterwards.
  */
 @AutoDelegate(typeToExtend = CassandraKeyValueService.class)
-public class CassandraKeyValueServiceImpl extends AbstractKeyValueService implements CassandraKeyValueService {
+public final class CassandraKeyValueServiceImpl extends AbstractKeyValueService implements CassandraKeyValueService {
```

Comment:
Is this  ok with the large internal product?

Reply:
Yep. We used to extend it for the TTL CassandraKVS, but we removed it in atlasdb recently.